Tools/ci-run.sh

```diff
@@ -119,9 +119,8 @@ else
 
   # Install more requirements
   if [[ $PYTHON_VERSION != *"-dev" ]]; then
-    if [[ $BACKEND == *"cpp"* ]]; then
-      echo "WARNING: Currently not installing pythran due to compatibility issues"
-      # python -m pip install pythran==0.9.5 || exit 1
+    if [[ $BACKEND == *"cpp"* && $OSTYPE != "msys" ]]; then
+      python -m pip install pythran || exit 1
```

Comment:
should you specify a minimal version here if some version were no longer working?

Reply:
Thanks. It isn't hugely easy to pin down a minimum version. I had a quick go and ended up lost in a mess of other dependencies not quite being compatible.

So I think I'll leave it as is - it's only for our CI and the version it picks is only likely to go up.